helium/config.json

```diff
@@ -28,33 +28,195 @@
      "//fonts.googleapis.com/css?family=Open+Sans:400italic,400,700"
   ],
   "extjs" : [],
-  "defaults": {
```

Comment:
the defaults are still translated to the same destination (theme.learn_data._defaults) by package-theme.py. so backwards compatibility is still maintained without keeping them here in helium/config.json